context_test.go

```diff
@@ -118,6 +113,51 @@ func TestMemoryLeak(t *testing.T) {
 	}
 }
 
+// https://github.com/rogchap/v8go/issues/186
+func TestRegistryFromJSON(t *testing.T) {
+	t.Parallel()
+
+	iso := v8.NewIsolate()
+	defer iso.Dispose()
+
+	global := v8.NewObjectTemplate(iso)
+	err := global.Set("location", v8.NewFunctionTemplate(iso, func(info *v8.FunctionCallbackInfo) *v8.Value {
+		v, err := v8.NewValue(iso, "world")
+		if err != nil {
+			t.Errorf("creating return value: %v", err)
+		}
```

Comment:
Other than the last assertion, all the others would just result in following test errors, so they may as well be fatal errors for the test.

There is a failIf helper that can be used instead to simplify these

```suggestion
		failIf(t, err)
```